protons/driver.py

```diff
@@ -587,6 +588,10 @@ def _set_titration_state(self, titration_group_index, titration_state_index, con
                             (titration_state_index, self._get_num_titration_states(titration_group_index)))
 
         self._update_forces(titration_group_index, titration_state_index, context=context)
+        # Update paramters in context
```

Comment:
I think the last step of `_update_forces` is to call `updateParametersInContext`. Is there a reason why it needs to be called again? (Or perhaps, did that part get changed?)